wiki/Submission/en.md

```diff
@@ -1,150 +1,31 @@
-
-Please see [Ranking Criteria](/wiki/Ranking_Criteria) for further information on this.
-
-## Questions
-
-### My map's not getting modded/ranked...
-
-There are far, *far* more beatmappers than moderators and, if *they* are sloppy, they can submit beatmaps faster than moderators can mod them. Furthermore, moderators do not only mod beatmaps as part of their duties - they also patrol the forums and chat, provide support to players, and (god forbid) *make* beatmaps and play osu! once in a while. This means that a key factor when submitting a beatmap is **patience**. Please be reminded that moderators also need to attend to their *physical life*, and does moderation out of hobby interest.
-
-Moderators will mod any beatmap, but it helps if your beatmap shows you've taken some effort and care into its creation.
-
-You can also try bumping your beatmap's thread via the Beatmap Submission System (use once every few days only) or going to osu's \#modding channel and requesting for your beatmap to be modded.
-
-After your beatmap receives initial suggestions, responding to them in a timely manner, showing determination on improving your beatmap for rank will help keep the attention of moderators to your beatmap.
-
-In case you receive comments and suggestions from users who are not moderators, you'd do well *not* to ignore them. While it's true that some of them may be incorrect, others could be just as useful as advice by moderators. In fact, such posts are ones that mark a user as a potential future moderator. Speaking of which, if you feel you can provide some feedback on beatmaps yourself, you'll indirectly help your own beatmap being modded, since you'll lighten the load on moderators.
-
-For further information, see [Modding](/wiki/Modding).
-
-### How many maps may I upload?
-
-All osu! users have a maximum number of unranked beatmaps in pending they are allowed to have.
-
-Depending on how many ranked maps you have, you are allowed between **3 to 6 unranked maps.** This number increases by one each time one of your maps is ranked. Only maps in Pending and Help forums are counted. The cap is in place to help control the flow of new beatmaps and to keep mappers more dedicated to their existing maps.
+Beatmaps will fail to submit if they exceed the online file size limit. That limit is 5MB plus an additional 10MB for every minute of beatmap length, and it caps at 100MB.
 
-**osu!supporters** have a larger cap and are allowed between **4 to 10 unranked maps.**
+Users can have a limited number of pending beatmaps at a time. The limit varies depending on how many ranked beatmaps a user has and whether or not they currently have [osu!supporter](https://osu.ppy.sh/help/wiki/osu!supporter). Users without osu!supporter can have 4 pending beatmaps plus 1 per ranked beatmap (up to 2). With osu!supporter, this increases to 8 pending beatmaps plus 1 per ranked beatmap (up to 12).
```

Comment:
This is wrong, supporter's max is 20 now. I don't know the particulars of how non-supporter or number of ranked beatmaps currently plays into this though.

Reply:
8 pending + 12 ranked map bonuses is 20? am i misinterpreting something

Reply:
O. The way it's  written comes off as 8 going up to a max of 12 (so capping after 4 ranked maps)

for a total of up to 20?